standard/standard-library.md

```diff
@@ -658,3 +652,400 @@ namespace System.Runtime.CompilerServices
+the percent
+symbol.</td>
+</tr>
+<tr>
+<td><p><code>E0</code></p>
+<p><code>E+0</code></p>
+<p><code>E-0</code></p>
+<p><code>e0</code></p>
+<p><code>e+0</code></p>
+<p><code>e-0</code></p></td>
+<td><strong>Engineering format:</strong> If any of the strings ‘E’, ‘E+’, ‘E-’, ‘e’, ‘e+’, or ‘e-’ are present
+in a custom format and is followed immediately by at least one ‘0’
+character, then the value is formatted using scientific notation. The number
+of ‘0’ characters following the exponent prefix (E or e) determines the
+minimum number of digits in the exponent. The ‘E+’ and ‘e+’ formats indicate
+that a positive or negative number symbol always precedes the
+exponent. The ‘E’, ‘E-’, ‘e’, or ‘e-’ formats indicate that a negative number symbol
+precedes negative exponents; no symbol is precedes positive exponents. The
+positive number symbol is supplied by the <code>System.Globalization.NumberFormatInfo.PositiveSign</code> property. The negative number symbol
+is supplied by the <code>System.Globalization.NumberFormatInfo.NegativeSign</code>
+
+property.</td>
+</tr>
+<tr>
+<td><code>\</code> (backslash)</td>
+<td><strong>Escape character:</strong> In some languages, such as C#, the
+backslash character causes the next character in the custom format to be interpreted
+as an escape sequence. It is used with C language
+formatting sequences, such as "\n" (newline). In some languages, the escape character
+itself is required to be preceded by an escape character
+when used as a literal. Otherwise, the compiler interprets the character as
+an escape sequence. This escape character is not required to be
+supported in all programming languages.</td>
+</tr>
+<tr>
+<td><p><code>'ABC'</code></p>
+<p><code>"ABC"</code></p></td>
+<td><strong>Literal string:</strong> Characters enclosed in single or double quotes are
+copied to the output string literally, and do not affect formatting.</td>
+</tr>
+<tr>
+<td><code>;</code> (semicolon)</td>
+<td><strong>Section separator:</strong> The ‘;’ character is used to separate sections for
+positive, negative, and zero numbers in the format string. (This feature
+is described in detail below.)</td>
+</tr>
+<tr>
+<td>Other</td>
+<td><strong>All other characters:</strong> All other characters are stored in the output
+string as literals in the position in which they
+appear.</td>
+</tr>
+</table>
+
+Note that for fixed-point format strings (strings not containing an ‘E0’,
+‘E+0’, ‘E-0’, ‘e0’, ‘e+0’, or ‘e-0’), numbers are rounded to as many decimal
+places as there are Zero or Digit placeholders to the right of the decimal
+separator. If the custom format does not contain a decimal separator, the number is
+rounded to the nearest integer. If the number has more digits than there are
+Zero or Digit placeholders to the left of the decimal separator, the extra
+digits are copied to the output string immediately before the first Zero or
+Digit placeholder.
+
+A custom format can contain
```

Comment:
Again, this is all about numeric format strings.

Reply:
Look two comments back